starter-code/src/clue.js

```diff
@@ -1,94 +1,227 @@
+  age: 36,
+  image: 'https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg',
+  occupation:'Socialité'
+}
+
+const mrMustard = {
+  first_name: 'Jack',
+  last_name: 'Mustard',
+  color: 'yellow',
+  description: 'He is a former football player who tries to get by on his former glory',
+  age: 62,
+  image: 'https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg',
+  occupation: 'Retired Football player'
+}
 
 // Weapons
 
-name: rope        ---   weight: 10
-name: knife       ---   weight: 8
-name: candlestick ---   weight: 2
-name: dumbbell    ---   weight: 30
-name: poison      ---   weight: 2
-name: axe         ---   weight: 15
-name: bat         ---   weight: 13
-name: trophy      ---   weight: 25
-name: pistol      ---   weight: 20
+const rope = {
+  name: 'rope',
+  weight: 10
+}
+const knife = {
+  name: 'knife',
+  weight: 8
+}  
+const candlestick = {
+  name: 'candlestick',
+  weight: 2
+}  
+const dumbbell = {
+  name: 'dumbbell',  
+  weight: 30
+}
+const poison = {
+  name: 'poison',  
+  weight: 2
+}  
+const axe = {
+  name: 'axe',  
+  weight: 15
+}
+const bat = {
+  name: 'bat',  
+  weight: 13
+}
+const trophy = {
+  name: 'trophy',  
+  weight: 25
+}
+const pistol = {
+  name: 'pistol',  
+  weight: 20
+}
 
 // Rooms
-name: Dining Room
-name: Conservatory
-name: Kitchen
-name: Study
-name: Library
-name: Billiard Room
-name: Lounge
-name: Ballroom
-name: Hall
-name: Spa
-name: Living Room
-name: Observatory
-name: Theater
-name: Guest House
-name: Patio
-*/
+const DiningRoom = {
+  name: 'DiningRoom',  
+}
+const Conservatory = {
+  name: 'Conservatory',  
+}
+const Kitchen = {
+  name: 'pisKitchentol',  
+}
+const Study = {
+  name: 'Study', 
+}
+const Library = {
+  name: 'Library',  
+}
+const BilliardRoom = {
+  name: 'BilliardRoom',  
+}
+const Lounge = {
+  name: 'Lounge',  
+}
+const Ballroom = {
+  name: 'Ballroom',  
+}
+const Hall = {
+  name: 'Hall',  
+}
+const Spa = {
+  name: 'Spa',  
+}
+const LivingRoom = {
+  name: 'LivingRoom',  
+}
+const Observatory = {
+  name: 'Observatory',  
+}
+const Theater = {
+  name: 'Theater',  
+}
+const GuestHouse = {
+  name: 'GuestHouse',  
+}
+const Patio = {
+  name: 'Patio',  
+}
 
 // Characters Collection
-const charactersArray = [];
 
+const charactersArray = [
+  mrGreen,
+  drOrchid,
+  profPlum,
+  missScarlet,
+  mrsPeacock,
+  mrMustard
+];
 // Rooms Collection
-const roomsArray = [];
+
+const roomsArray = [
+  DiningRoom,
+  Conservatory,
+  Kitchen,
+  Study,
+  Library,
+  BilliardRoom,
+  Lounge,
+  Ballroom,
+  Hall,
+  Spa,
+  LivingRoom,
+  Observatory,
+  Theater,
+  GuestHouse,
+  Patio,
+];
 
 // Weapons Collection
-const weaponsArray = [];
+const weaponsArray = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+];
+
+// iteración 2
+
+//random selector
+function randomSelector (array) {
+  if (array.length === 0) {
+    return undefined
```

Comment:
Mejor si negáis el resultado `truthy` de la condición. `if(!array.length) return undefined`